doc/release_notes/release_notes_2.2.rst

```diff
@@ -0,0 +1,118 @@
+.. _release_notes_2.2:
+
+ACRN v2.1 (Sep 2020)
+####################
+
+We are pleased to announce the release of ACRN version 2.2
+
+
+ACRN is a flexible, lightweight reference hypervisor, built with real-time and safety-criticality in mind, optimized to streamline embedded development through an open source platform. Check out the What is ACRN for more information. All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor repository and includes folders for the ACRN hypervisor, the ACRN device model, tools, and documentation. You can either download this source code as a zip or tar.gz file (see the ACRN v2.2 GitHub release page) or use Git clone and checkout commands:
+
+git clone https://github.com/projectacrn/acrn-hypervisor
+cd acrn-hypervisor
+git checkout v2.2
+The project’s online technical documentation is also tagged to correspond with a specific release: generated v2.2 documents can be found at https://projectacrn.github.io/2.2/. Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/. ACRN v2.2 requires Ubuntu 18.04. Please follow the instructions in the Getting Started Guide for ACRN Industry Scenario with Ubuntu Service VM.
+
+
+
+Version 2.2 major features
+What’s New in v2.2
+
+
+Elkhart Lake and Tiger Lake processor support: At Intel Industrial Summit 2020, Intel announced the latest additions to their enhanced-for-IoT Edge portfolio: the Intel® Atom® x6000E Series, Intel® Pentium® and Intel®Celeron® N and J Series (all codenamed Elkhart Lake), and 11th Gen Intel® Core™ processors (codename Tiger Lake-UP3). The ACRN team is pleased to announce that this ACRN v2.2 release already supports these processors.
+
+Support for time deterministic applications with new features e.g., Time Coordinated Computing and Time Sensitive Networking
+Support for functional safety with new features e.g., Intel Safety Island
+ACRN can boot on Slim Bootloader on Elkhart Lake, an alternative bootloader to UEFI BIOS.
+
+Shared memory based inter-VM communication (ivshmem) is now extended to support all kinds of VMs including pre-launched VM, Service VM and other guest VMs.
+
+CPU sharing supports pre-launched VM
+
+RTLinux with preempt-RT linux kernel 5.4 is validated as a pre-launched VM as well as a post-launched VM.
+
+ACRN hypervisor can emulate MSI-X based on physical MSI with multiple vectors.
+
+Staged removal of deprivileged boot mode support: ACRN used deprivileged boot mode to meet Clearlinux’s special requirement, which is actually bringing scalability effort and not suitable for hybrid mode support. So ACRN is moving to support Grub/multiboot2, a much simpler way, which is also more scalable. Consequently, depriviledged boot mode is no longer the default in ACRN v2.2 and will be completely removed in v2.3
```

Comment:
@dbkinder can we rephrase this along those lines?

"ACRN has supported deprivileged boot mode to ease the integration in Linux distributions such as Clearlinux. It does however bring scalability challenges. We are therefore removing support for deprivileged boot mode to focus on multiboot2 boot more. multiboot2 is not supported in Clearlinux so we have made the decision to focus on Ubuntu as the Service VM OS moving forward, it supports multiboot2 via Grub (its default bootloader)."

The rest about the versions, etc. is fine.